lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -732,6 +732,306 @@ class DecomposeAtenTriuOp : public OpRewritePattern<AtenTriuOp> {
 };
 } // namespace
 
+Value arange(PatternRewriter &rewriter, Location loc, Value size, Value dtype,
+             Value layout, Value device, Value pin_memory, Type type) {
+
+  auto arrangeType = getTensorTypeFromShapeValues({size}, type);
+  return rewriter.create<AtenArangeOp>(loc, arrangeType, size,
+                                       /*dtype=*/dtype, /*layout=*/layout,
+                                       /*device=*/device,
+                                       /*pin_memory=*/pin_memory);
+}
+
+void _getTrilSizes(PatternRewriter &rewriter, Location loc, int64_t row,
+                   int64_t col, int64_t offset,
+                   /*return values*/ int64_t &trapezoidSize,
+                   int64_t &rectangleSize) {
```

Comment:
nit: passing a reference to the result values is usually useful when the function returns an error message, but that is not the case here. I find it more readable to return a `std::pair` of `int`s, and avoid having implicit mutation of inputs to the function.

Reply:
Instead of `std::pair` I used `std::tuple`, because functions `getTriuSizes `and `getTrilSizes `have 3 return values.